src/Uno.UI/UI/Xaml/Media/SolidColorBrush.cs

```diff
@@ -127,5 +127,9 @@ public bool Equals(SolidColorBrush other)
 		public static bool operator ==(SolidColorBrush left, SolidColorBrush right) => Equals(left, right);
 
 		public static bool operator !=(SolidColorBrush left, SolidColorBrush right) => !Equals(left, right);
+
+		public static implicit operator Color(SolidColorBrush scb) => scb?.Color ?? Colors.Transparent;
```

Comment:
This definitely shouldn't be added. It's not part of the WinUI contract.

Reply:
It has been converted to an `explicit` operator instead. It's required for generated code to compile with WinUI patterns (using a `SolidColorBrush` to a `Color?` property).

Reply:
Can you point to the Xaml for which this is necessary? It seems that it's valid somehow to template-bind a ColorAnimation.To/From property to a SolidColorBrush, but in a quick test it doesn't seem to be allowed to assign a SCB StaticResource directly.

Eg, this xaml:

```xaml
		<SolidColorBrush x:Key="VeryGoodBrush"
						 Color="Red" />
		<Storyboard x:Name="yeStoryboard">
			<ColorAnimation Storyboard.TargetName="TargetBorder"
							Storyboard.TargetProperty="(Border.Background).(SolidColorBrush.Color)"
							From="Blue"
							To="{StaticResource VeryGoodBrush}"
							/>
		</Storyboard>
```

gives:
![image](https://user-images.githubusercontent.com/8270914/81765470-e7556400-94a1-11ea-8f88-1d1519fb4828.png)

Reply:
@davidjohnoliver https://github.com/microsoft/microsoft-ui-xaml/blob/master/dev/ProgressBar/ProgressBar.xaml#L44_L48

Foreground, which is a `Brush` is templated-bound to a `Color?` property.

Reply:
Right, which will create a binding. There's no compile-time type checking for a binding. It should go through the FastConverter you added, which is why I don't understand why there's additionally a need for the user-defined conversion operator.

Reply:
Fixed.